<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="CmdRichtung">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>(Sprachauswahl treffen)</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdTestStarten">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test starten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdTestBeenden">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test beenden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdPruefen">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Prüfen / Nächster</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdNeuEingabeStarten">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Neueingabe starten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdNeuEingabeSpeichern">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Neueingabe speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdAuswahlLoeschen">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Auswahl löschen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdNeuEingabeBeenden">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Neueingabe beenden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtDeu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtEng">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtFra">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="LstDeu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>201</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="LstEng">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="LstFra">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtAntwort">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LstLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>26</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
